core/trino-main/pom.xml

```diff
@@ -579,6 +579,17 @@
                             <installDirectory>src/main/resources/webapp/dist</installDirectory>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>flow check</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <configuration>
+                            <arguments>run flow</arguments>
+                            <workingDirectory>src/main/resources/webapp/src</workingDirectory>
+                            <installDirectory>src/main/resources/webapp/dist</installDirectory>
```

Comment:
should this be somewhere in `target`?

Reply:
not really. it will be packaged from this directory into the jar. it's ignored in the .gitignore so won't be checked in to git

Reply:
I've checked that most of the projects using frontend-maven-plugin follow that approach